### PR TITLE
updating location of logic for getmail command

### PR DESCRIPTION
**Pull request by modernNeo**
moving logic for processing all the new emails from all mailboxes to a method under Mailbox so it can be called from other modules

### CODE REVIEW

File: django_mailbox/management/commands/getmail.py [hunk 1]
```diff
@@ -4,27 +4,10 @@
 
 from django_mailbox.models import Mailbox
 
-
 logger = logging.getLogger(__name__)
```
Comment (coddingtonbear) [inline django_mailbox/management/commands/getmail.py:7]: I know this is where _I_ put this, but could I convince you into moving these two logger lines into the `handle` method below so importing this module is side-effect free?

Comment (modernNeo) [reply django_mailbox/management/commands/getmail.py]: done with f531c7865bfcec8d62f01af1a03563dcfd48cb57

**Comment by modernNeo** [comment]
@coddingtonbear thanks for merging 😄 

Do I need to submit another PR to bump the versions or will you do that yourself?

**Comment by coddingtonbear** [comment]
No, no, I can sort that out!  Thanks for taking the time to put together a patch, @modernNeo!

**Comment by modernNeo** [comment]
Hi @coddingtonbear ,

when do you think you will get around to releasing the patch?

**Pull request merged**